Build error: unable to open image `portal_pak_dir/materials/models/props/round_elevator_sheet_3.png

I've been using the docker image to build. Can't seem to do it anymore. Tried deleting everything, cloned the repo, pruned my docker images and containers. VPK files are all there. Getting this in the console:

```
convert portal_pak_dir/materials/models/props/elevator_caps/elevator_caps.png -alpha off     -crop 498x102+20+606 -resize 128x portal_pak_modified/materials/models/props/elevator_caps/elevator_caps.png
vtf2png portal_pak_dir/materials/models/props/round_elevator_sheet_1.vtf portal_pak_dir/materials/models/props/round_elevator_sheet_1.png
convert portal_pak_dir/materials/models/props/round_elevator_sheet_1.png -alpha off      \( +clone -crop 280x280+1550+985 -resize 32x32 -fill "#262d2f" -draw 'rectangle 0,-1 32,0' -draw 'rectangle 0,31 32,32' -draw 'rectangle -1,0 0,32' -draw 'rectangle 31,0 32,32' -write portal_pak_modified/materials/models/props/round_elevator_floor.png +delete \)     -crop 162x554+1031+26     +repage     \( portal_pak_dir/materials/models/props/round_elevator_sheet_3.png -crop 14x480+1170+1051 -repage +0+0 \)     -compose Over     -layers merge     -resize 16x128 portal_pak_modified/materials/models/props/round_elevator_sheet_1.png
convert-im6.q16: unable to open image `portal_pak_dir/materials/models/props/round_elevator_sheet_3.png': No such file or directory @ error/blob.c/OpenBlob/2924.
convert-im6.q16: geometry does not contain image `portal_pak_dir/materials/models/props/round_elevator_sheet_1.png' @ warning/transform.c/CropImage/668.
make: *** [Makefile:134: portal_pak_modified/materials/models/props/round_elevator_sheet_1.png] Error 1
make: *** [Makefile.docker:2: all] Error 2
```

I was able to compile but only if I disabled -Werror otherwise I got these errors/warnings. When successfully compiling with -Werror disabled, I'm stuck in the first chamber. Clock stuck at zero, no cutscene, no portal.

```
In file included from build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:1:
build/assets/test_chambers/test_chamber_00/test_chamber_00.h:57:63: error: initialization of 'struct ObjectTriggerInfo *' from 'int' makes pointer from integer without a cast [-Werror=int-conversion]
   57 | #define TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE 1
      |                                                               ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12789:9: note: in expansion of macro 'TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE'
12789 |         TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
build/assets/test_chambers/test_chamber_00/test_chamber_00.h:57:63: note: (near initialization for 'test_chamber_00_test_chamber_00_triggers[1].triggers')
   57 | #define TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE 1
      |                                                               ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12789:9: note: in expansion of macro 'TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE'
12789 |         TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_FIRST_PUZZLE,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
build/assets/test_chambers/test_chamber_00/test_chamber_00.h:58:68: error: initialization of 'struct ObjectTriggerInfo *' from 'int' makes pointer from integer without a cast [-Werror=int-conversion]
   58 | #define TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM 2
      |                                                                    ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12797:9: note: in expansion of macro 'TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM'
12797 |         TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
build/assets/test_chambers/test_chamber_00/test_chamber_00.h:58:68: note: (near initialization for 'test_chamber_00_test_chamber_00_triggers[2].triggers')
   58 | #define TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM 2
      |                                                                    ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12797:9: note: in expansion of macro 'TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM'
12797 |         TEST_CHAMBER_00_TEST_CHAMBER_00_CUTSCENE_START_SECOND_ROOM,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12805:9: error: initialization of 'struct ObjectTriggerInfo *' from 'int' makes pointer from integer without a cast [-Werror=int-conversion]
12805 |         -1,
      |         ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12805:9: note: (near initialization for 'test_chamber_00_test_chamber_00_triggers[3].triggers')
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12813:9: error: initialization of 'struct ObjectTriggerInfo *' from 'int' makes pointer from integer without a cast [-Werror=int-conversion]
12813 |         -1,
      |         ^
build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.c:12813:9: note: (near initialization for 'test_chamber_00_test_chamber_00_triggers[4].triggers')
cc1: all warnings being treated as errors
make: *** [Makefile:303: build/assets/test_chambers/test_chamber_00/test_chamber_00_geo.o] Error 1
```

Resolved the -Werror/cutscenes issues by updating my docker image. Whoops! 

My issues are mostly resolved, but vpk building still has some issues and elevator textures are missing in game.  Here are my steps I currently have to to do:
- Start image in interactive mode, because first pass of VPK assets will fail.
- Run `make clean` and `rm -rf portal_pak_*` for good measure.
- Run `make`
  - Get first error (See error 1)
- Run `make` again
  - Get different error (See error 2)
- Run `./skelatool64/skeletool64 --name ui --default-material default_ui -m assets/materials/ui.skm.yaml --material-output -o build/assets/materials/ui.h`
- Run `make` one last time
  - Build succeeds 🎉
- Transfer ROM to cart via UNFLoader
  - Works as expected, but missing elevator textures 🛗 

Error 1
```
convert portal_pak_dir/materials/signage/indicator_lights/indicator_lights_floor.png -alpha off -crop 64x64+0+0 -resize 16x16 \( portal_pak_dir/materials/signage/indicator_lights/indicator_lights_corner_floor.png -alpha off -resize 16x16 -repage +0+16 \) -compose Copy -layers merge -alpha Activate \(     \( portal_pak_dir/materials/signage/indicator_lights/indicator_lights_floor.png -alpha extract -crop 64x64+0+0 -resize 16x16 -repage +0+0 \)     \( portal_pak_dir/materials/signage/indicator_lights/indicator_lights_corner_floor.png -alpha extract -resize 16x16 -repage +0+16 \)     -compose Copy     -layers merge \) -compose CopyOpacity -composite portal_pak_modified/materials/signage/indicator_lights/indicator_lights_floor.png
convert-im6.q16: unable to open image `portal_pak_dir/materials/signage/indicator_lights/indicator_lights_corner_floor.png': No such file or directory @ error/blob.c/OpenBlob/2924.
make: *** [Makefile:134: portal_pak_modified/materials/signage/indicator_lights/indicator_lights_floor.png] Error 1
  ```
  
 Error 2
 ```
 convert portal_pak_dir/materials/models/props/round_elevator_sheet_1.png -alpha off      \( +clone -crop 280x280+1550+985 -resize 32x32 -fill "#262d2f" -draw 'rectangle 0,-1 32,0' -draw 'rectangle 0,31 32,32' -draw 'rectangle -1,0 0,32' -draw 'rectangle 31,0 32,32' -write portal_pak_modified/materials/models/props/round_elevator_floor.png +delete \)     -crop 162x554+1031+26     +repage     \( portal_pak_dir/materials/models/props/round_elevator_sheet_3.png -crop 14x480+1170+1051 -repage +0+0 \)     -compose Over     -layers merge     -resize 16x128 portal_pak_modified/materials/models/props/round_elevator_sheet_1.png
convert-im6.q16: unable to open image `portal_pak_dir/materials/models/props/round_elevator_sheet_3.png': No such file or directory @ error/blob.c/OpenBlob/2924.
convert-im6.q16: geometry does not contain image `portal_pak_dir/materials/models/props/round_elevator_sheet_1.png' @ warning/transform.c/CropImage/668.
make: *** [Makefile:134: portal_pak_modified/materials/models/props/round_elevator_sheet_1.png] Error 1
 ```

--- a/Makefile
+++ b/Makefile
@@ -64,7 +64,12 @@ LDIRT  =	$(BASE_TARGET_NAME).elf $(CP_LD_SCRIPT) $(BASE_TARGET_NAME).z64 $(BASE_
 
 LDFLAGS =	-L/usr/lib/n64 $(N64LIB)  -L$(N64_LIBGCCDIR) -lgcc
 
-default:	$(BASE_TARGET_NAME).z64
+default:	prepare
+
+prepare: portal_pak_dir
+	@$(MAKE) buildgame
+
+buildgame: $(BASE_TARGET_NAME).z64
 
 include $(COMMONRULES)
 
@@ -121,7 +126,7 @@ $(TEXTURE_VTF_SOURCES): portal_pak_dir
 
 %.png: %.vtf
 	-$(VTF2PNG) $< $@
-	
+
 portal_pak_dir/materials/signage/signage_doorstate_on.png: portal_pak_dir/materials/signage/signage_doorstate.vtf
 	$(VTF2PNG) -f 2 $< $@
 
@@ -133,6 +138,12 @@ portal_pak_dir/materials/signage/indicator_lights/indicator_lights_floor_on.png:
 
 portal_pak_dir/materials/effects/portal_1_particle_orange.png: portal_pak_dir/materials/effects/portal_1_particle.vtf
 	$(VTF2PNG) -f 2 $< $@
+
+portal_pak_dir/materials/signage/signage_overlay_fling1.png: portal_pak_dir/materials/signage/signage_overlay_fling2.png portal_pak_dir/materials/signage/signage_overlay_dots1.png portal_pak_dir/materials/signage/signage_overlay_dots2.png portal_pak_dir/materials/signage/signage_overlay_dots3.png portal_pak_dir/materials/signage/signage_overlay_dots4.png portal_pak_dir/materials/signage/signage_overlay_toxic.png portal_pak_dir/materials/signage/signage_overlay_fountain.png
+portal_pak_dir/materials/signage/signage_exit.png: portal_pak_dir/materials/signage/signage_overlay_arrow.png portal_pak_dir/materials/signage/signage_overlay_boxdispenser.png portal_pak_dir/materials/signage/signage_overlay_boxhurt.png portal_pak_dir/materials/signage/signage_overlay_energyball.png portal_pak_dir/materials/signage/signage_overlay_catcher.png portal_pak_dir/materials/signage/signage_overlay_toxic.png portal_pak_dir/materials/signage/signage_overlay_fountain.png
+portal_pak_dir/materials/signage/indicator_lights/indicator_lights_floor.png: portal_pak_dir/materials/signage/indicator_lights/indicator_lights_corner_floor.png
+portal_pak_dir/materials/signage/indicator_lights/indicator_lights_floor_on.png: portal_pak_dir/materials/signage/indicator_lights/indicator_lights_corner_floor_on.png
+portal_pak_dir/materials/models/props/round_elevator_sheet_1.png: portal_pak_dir/materials/models/props/round_elevator_sheet_3.png
 
 convert_all_png: $(ALL_PNG_IMAGES)
 
@@ -251,6 +262,7 @@ build/src/decor/decor_object_list.o: build/assets/models/dynamic_model_list.h bu
 build/src/effects/effect_definitions.o: build/assets/materials/static.h
 build/src/menu/controls.o: build/assets/materials/ui.h build/src/audio/clips.h
 build/src/menu/game_menu.o: build/src/audio/clips.h build/assets/materials/ui.h build/assets/materials/images.h build/assets/test_chambers/test_chamber_00/test_chamber_00.h
+build/src/menu/gameplay_options.o: build/assets/materials/ui.h build/src/audio/clips.h
 build/src/menu/joystick_options.o: build/assets/materials/ui.h build/src/audio/clips.h
 build/src/menu/landing_menu.o: build/assets/materials/ui.h build/src/audio/clips.h
 build/src/menu/load_game.o: build/assets/materials/ui.h build/src/audio/clips.h
@@ -449,6 +461,8 @@ $(BASE_TARGET_NAME)_debug.z64: $(CODESEGMENT)_debug.o $(OBJECTS) $(DATA_OBJECTS)
 
 clean:
 	rm -rf build
+	rm -rf portal_pak_dir
+	rm -rf portal_pak_modified		
 
 fix:
 	wine tools/romfix64.exe build/portal.z64 

